maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml

```diff
@@ -27,7 +27,8 @@
 #     - policy name 2
 
 blueprint:
-subscription:
+  - Volume
```

Comment:
lets set this to [] as well?

Reply:
@Michaelpalacce it doesn't work with empty blueprints as we have tested with @joroaf the newly created project from the archetype

Reply:
@imihov1988 You get an error when pulling empty blueprints? really?

Reply:
@Michaelpalacce  Yes, it was not erroring out when trying to build it.